jobs/dd-agent/templates/config/datadog.yaml.erb

```diff
@@ -59,11 +58,10 @@ config_providers:
     polling: true
 
 cluster_agent:
-  auth_token: "<%= dca_link.p("cluster_agent.token") %>"
-  url: https://<%= dca_link.address %>:<%= dca_link.p("cluster_agent.port") %>
-  enabled: <%= dca_link.p("cluster_agent.enabled") %>
+  auth_token: "<%= p("cluster_agent.token") %>"
+  url: https://datadog-cluster-agent.cf.internal:<%= p("cluster_agent.port", 5005) %>
```

Comment:
Should we hardcode this in here or make it configurable by the user ?
Also any better ideas for the DNS name ?